index.src.html

```diff
@@ -548,6 +548,34 @@ <h1>Credential Management Level 1</h1>
     to guide the discovery process (by choosing a federated identity provider, BTLE device, etc).
   </div>
 
+  <div algorithm="copy any buffer sources">
+    To <dfn>create a copy of any buffer sources' data</dfn> referenced by a value |V| of
```

Comment:
The following reads a little more clearly to me:

```suggestion
    To <dfn>create a copy of any buffer sources' data</dfn> referenced by a [=dictionary=] value |V|, run ...
```